pkgs/development/libraries/jemalloc/default.nix

```
@@ -48,10 +51,14 @@ stdenv.mkDerivation rec {
     autogen
     autoconf
     automake
+    git
```

```suggestion
    gitMinimal
```
We probably don't need ssh support here, so we can reduce dependencies

---

thx, updated!